pkg/apis/crd/v1alpha1/types.go

```diff
@@ -549,3 +549,103 @@ type PacketCaptureCondition struct {
 	Reason             string                     `json:"reason"`
 	Message            string                     `json:"message"`
 }
+
+type FlowExporterIPFIXTransportProtocol string
+
+const (
+	ProtoTCP FlowExporterIPFIXTransportProtocol = "tcp"
+	ProtoUDP FlowExporterIPFIXTransportProtocol = "udp"
+	ProtoTLS FlowExporterIPFIXTransportProtocol = "tls"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type FlowExporterDestination struct {
```

Comment:
Maybe we should follow the latest guidance for K8s APIs:

_new fields should explicitly set either an +optional or +required comment tag._

Even though in our case, I don't know if any tool we use actually uses these tags.

Reply:
Updated with `+optional` and `+required`